app/src/main/res/values/donottranslate.xml

```diff
@@ -12,8 +12,10 @@
     <item name="customize_fonts" type="string" translatable="false">customize_fonts</item>
     <item name="font_sizes" type="string" translatable="false">font_sizes</item>
     <item name="dhis_url" type="string" translatable="false">dhis_url</item>
+    <item name="dhis_user" type="string" translatable="false">dhis_user</item>
+    <item name="dhis_password" type="string" translatable="false">dhis_password</item>
     <item name="org_unit" type="string" translatable="false">org_unit</item>
-
+    <item name="pull_metadata" type="string" translatable="false">pull_metadata</item>
```

Comment:
@Strabagkantza as we have just talked, we would also need to:
- [ ] Add pull to login (only data)
- [ ] Remove Login from settings
- [ ] Add Logout to settings
- [ ] Add EULA acceptance to Login